package.json

```diff
@@ -122,7 +122,9 @@
     ]
   },
   "dependencies": {
+    "@meetfranz/electron-notification-state": "^1.0.0",
     "@paulcbetts/system-idle-time": "^1.0.4",
+    "@sindresorhus/do-not-disturb": "^0.2.0",
```

Comment:
we are not using this anymore, right?

Reply:
Yes we don't use it, I deprecated it :-)